### ISSUE TITLE
Disable the showing of required taxonomy on published event

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#5212]
Disable the showing of required taxonomy on published event

**Comment by RichieB2B** [comment]
Perhaps make this optional? I think it’s useful to know a published event is missing a required tag.

**Comment by adulau** [comment]
Indeed, the idea was to not show it for event which don't have ownership (not the same organisation).
